docs/graphx-programming-guide.md

```diff
@@ -720,25 +722,53 @@ class GraphOps[VD, ED] {
        sendMsg: EdgeTriplet[VD, ED] => Iterator[(VertexId, A)],
        mergeMsg: (A, A) => A)
     : Graph[VD, ED] = {
-    // Receive the initial message at each vertex
-    var g = mapVertices( (vid, vdata) => vprog(vid, vdata, initialMsg) ).cache()
+    val checkpointInterval = graph.vertices.sparkContext.getConf
```

Comment:
hmm, as this is just a implementation sketch, I don't think we should include such details of checkpointer.